cassandra/README.md

```diff
@@ -7,343 +7,149 @@
-#
+**Warning:** [Minikube](https://kubernetes.io/docs/docs/getting-started-guides/minikube/) defaults to 1024MB of memory and 1 CPU which results in an insufficient resource errors. 
 
-kubectl create -f cassandra/cassandra-daemonset.yaml --validate=false
+To avoid these errors, run minikube with:
 
-# resource cleanup
-kubectl delete service -l app=cassandra
-kubectl delete daemonset cassandra
 ```
+minikube start --memory 5120 --cpus=4
+``` 
+{% endcapture %}
 
-## Step 1: Create a Cassandra Headless Service
-
-A Kubernetes _[Service](/docs/user-guide/services)_ describes a set of
-[_Pods_](/docs/user-guide/pods) that perform the same task. In
-Kubernetes, the atomic unit of an application is a Pod: one or more containers
-that _must_ be scheduled onto the same host.
+{% capture lessoncontent %}
+## Creating a Cassandra Headless Service
+A Kubernetes [Service](/docs/concepts/services-networking/service/) describes a set of [Pods](https://kubernetes.io/docs/docs/concepts/workloads/pods/pod/) that perform the same task. 
 
-The Service is used for DNS lookups between Cassandra Pods, and Cassandra clients
-within the Kubernetes Cluster.
+The following `Service` is used for DNS lookups between Cassandra pods and clients within the Kubernetes Cluster.
 
-Here is the service description:
+1. `cd` to the folder you saved the .yaml files.
+2. Create a `Service` to track all Cassandra StatefulSet Nodes from the following `.yaml` file:
 
-<!-- BEGIN MUNGE: EXAMPLE cassandra-service.yaml -->
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: cassandra
-  name: cassandra
-spec:
-  clusterIP: None
-  ports:
-    - port: 9042
-  selector:
-    app: cassandra
+```shell
+kubectl create -f cassandra-service.yaml
 ```
 
-[Download example](https://raw.githubusercontent.com/kubernetes/examples/master/cassandra-service.yaml)
-<!-- END MUNGE: EXAMPLE cassandra-service.yaml -->
+{% include code.html language="yaml" file="cassandra-service.yaml" ghlink="/docs/tutorials/stateful-application/cassandra-service.yaml" %}
 
-Create the service for the StatefulSet:
+### Validating (optional)
 
+Get the Cassandra `Service`.
 
-```console
-$ kubectl create -f cassandra/cassandra-service.yaml
+```shell
+kubectl get svc cassandra
 ```
 
-The following command shows if the service has been created.
-
-```console
-$ kubectl get svc cassandra
-```
-
-The response should be like:
+The response should be
 
 ```console
 NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
 cassandra   None         <none>        9042/TCP   45s
 ```
 
-If an error is returned the service create failed.
-
-## Step 2: Use a StatefulSet to create Cassandra Ring
+If anything else returns, the service was not successfully created. Read [Debug Services](https://kubernetes.io/docs/docs/tasks/debug-application-cluster/debug-service/) for common issues.
 
-StatefulSets (previously PetSets) are a feature that was upgraded to a <i>Beta</i> component in
```

Comment:
can we keep this paragraph somehow? It highlights how Statefulsets are useful and links to the docs.